client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -257,22 +264,42 @@ class DisconnectJetpack extends PureComponent {
 					className="disconnect-jetpack__more-info-link"
 					href="https://jetpack.com/features/"
 				>
-					{ translate( 'Read More about Jetpack benefits' ) }
+					{ translate( 'Read more about Jetpack benefits' ) }
 				</a>
-			</Card>
-		);
+			</Card>,
+			'active' === rewindState && (
+				<Card
+					key="disconnect-jetpack__try-rewind"
+					className="disconnect-jetpack__try-rewind disconnect-jetpack"
+				>
+					<p className="disconnect-jetpack__highlight">
+						{ translate( 'Experiencing connection issues? Try to go back and rewind your site.' ) }
+					</p>
+					<div className="disconnect-jetpack__try-rewind-button-wrap">
+						<Button href={ `/stats/activity/${ siteSlug }` } onClick={ this.props.trackTryRewind }>
+							{ translate( 'Rewind site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Rewind Site' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).